packages/mui-material/src/Button/Button.js

```diff
@@ -76,189 +60,218 @@ const ButtonRoot = styled(ButtonBase, {
       ownerState.fullWidth && styles.fullWidth,
     ];
   },
-})(
-  ({ theme, ownerState }) => {
-    const inheritContainedBackgroundColor =
-      theme.palette.mode === 'light' ? theme.palette.grey[300] : theme.palette.grey[800];
+})(({ theme }) => {
+  const inheritContainedBackgroundColor =
```

Comment:
Shouldn't we use `applyStyles` instead of these two consts?

Reply:
No need to. This line will work for both Emotion (with or without CssVarsProvider) and Pigment.